### ISSUE TITLE
`prefer-to-be` auto fix breaks code when using `.resolves`

### ISSUE THREAD

**Issue by sapphi-red** [jest-community/eslint-plugin-jest#979]
### code
```ts
await expect(p).resolves.toBe(undefined)
```

`prefer-to-be` auto fix breaks this code.

### actual
```ts
await expect(p).toBeUndefined()
```

### expected
```ts
await expect(p).resolves.toBeUndefined()
```

**Comment by sapphi-red** [comment]
Thank you for your quick fix!
